README.md

```diff
@@ -73,6 +73,25 @@ Whenever the same functionality can be achieved in multiple offerings but in a d
 
 For links within this documentation, please end the link with `/index.md`. This way all links will be checked, and you’ll get an error if a link is broken at any point.
 
+### Concepts
+
+There are several key concepts at Snowplow: events (self-describing, structured), entities, schemas. We must ensure that we use and explain them consistently.
+
+**Please, use up-to-date terms:**
+* “Self-describing event”, not “unstructured event”
+* “Entities”, not “contexts”
+* If you are writing about schemas, pick “schema” or “data structure” and stick with it
+
+**Please, do not over-explain these in any of your writing.** Instead, just link to one of the existing concept pages:
+* Events in general: `/docs/understanding-your-pipeline/events/index.md`
+* Custom events: `/docs/understanding-your-pipeline/events/index.md#self-describing-events`
```

Comment:
The link is not a copy-paste error. In most cases where custom events are mentioned, the context (pun not intended) is that schemas are required. And also we recommend these over structured events.